arangod/Cluster/ClusterMethods.cpp

```diff
@@ -873,7 +886,7 @@ futures::Future<OperationResult> revisionOnCoordinator(
   for (auto const& p : *shards) {
     auto future = network::sendRequestRetry(
         pool, "shard:" + p.first, fuerte::RestVerb::Get,
-        "/_api/collection/" + StringUtils::urlEncode(p.first) + "/revision",
+        "/_api/collection/" + StringUtils::urlEncode(std::string{p.first}) + "/revision",
```

Comment:
If we are using ShardIDs here only, we can avoid the URL-encoding, because ShardIDs are always URL-safe.
This is true for this occurrence and all the following ones where we only refer to ShardIDs, not collection names.

Reply:
All done, they are all safe.